onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -3230,6 +3230,11 @@ void RegisterContribSchemas() {
           "(Optional) SDK version used to convert the model.",
           AttributeProto::STRING,
           OPTIONAL_VALUE)
+      .Attr(
+          "hardware_arch",
+          "(Optional) Hardware architecture of the GPU.",
```

Comment:
change to hardware specific string.  (remove mention of GPU)

Reply:
Changed.